docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -104,120 +111,123 @@ Set the following environment variables before running Ansible in order to confi
-
-Executing ``ansible`` or ``ansible-playbook`` and specifying the ``inventory`` directory instead of an individual file will cause ansible to evaluate each file in that directory for inventory.
-
-Let's once again use our inventory script to see if it can talk to Google Cloud:
-
-.. code-block:: bash
-
-    $ ansible all -i inventory/ -m setup
-    hostname | success >> {
-      "ansible_facts": {
-        "ansible_all_ipv4_addresses": [
-            "x.x.x.x"
-        ],
-
-The output should be similar to the previous command.  If you're wanting less output and just want to check for SSH connectivity, use "-m" ping instead.
-
-Use Cases
----------
-
-For the following use case, let's use this small shell script as a wrapper.
-
-.. code-block:: bash
-
-  #!/usr/bin/env bash
-  PLAYBOOK="$1"
+To use the gcp_compute inventory plugin, create a file that ends in .gcp.yml file in your root directory. The gcp_compute inventory script takes in the same authentication
+information as any module.
 
-  if [[ -z $PLAYBOOK ]]; then
-    echo "You need to pass a playbook as argument to this script."
-    exit 1
-  fi
+Here's an example of a valid inventory file:
 
-  export SSL_CERT_FILE=$(pwd)/cacert.pem
-  export ANSIBLE_HOST_KEY_CHECKING=False
+.. code-block:: yaml
 
-  if [[ ! -f "$SSL_CERT_FILE" ]]; then
-    curl -O http://curl.haxx.se/ca/cacert.pem
-  fi
+    plugin: gcp_compute
+    projects:
+      - google.com:graphite-playground
+    filters:
+    auth_kind: serviceaccount
+    service_account_file: /home/alexstephen/my_account.json
 
-  ansible-playbook -v -i inventory/ "$PLAYBOOK"
 
+Executing ``ansible-inventory --list -i <filename>.gcp.yml`` will create a list of GCP instances that are ready to be configured using ANsible.
 
 Create an instance
 ``````````````````
 
-The GCE module provides the ability to provision instances within Google Compute Engine. The provisioning task is typically performed from your Ansible control server against Google Cloud's API.
+The full range of GCP modules provide the ability to create a wide-variety of
+GCP resources with the full support of the entire GCP API.
 
-A playbook would looks like this:
+The following playbook creates a GCE Instance. This instance relies on a GCP
+network and a Disk. By creating the Disk and Network separately, we can give as
+much detail as necessary about how we want the disk and network formatted. By
+registering a Disk/Network to a variable, we can simply insert the variable
+into the instance task. The gcp_compute_instance module will figure out the
+rest.
 
 .. code-block:: yaml
 
-   - name: Create instance(s)
+   - name: Create instance.
```

Comment:
the period at the end of the play name looks weird to me

Reply:
Done.